apps/core/src/migration/service/convert-monitor-to-app-definition.spec.ts

```diff
@@ -11,10 +11,11 @@ const createMonitorChartWidget = (
   widgetType: MonitorWidgetType,
   metrics: MonitorMetric[],
   annotations?: MonitorAnnotations,
+  title?: string,
 ) => {
   return {
     type: widgetType,
-    title: 'test',
+    title: title ? title : 'test',
```

Comment:
title ?? 'test'

Reply:
Fixed in new PR: https://github.com/awslabs/iot-application/pull/1819